base/io.jl

```diff
@@ -510,6 +510,14 @@ line before it is returned. When `keep` is true, they are returned as part of th
 line.
 
 # Examples
+```julia-repl
```

Comment:
Does this example need to go first? I would probably recommend putting it below the existing examples, unless it really needs to be at the top.